nomad/drainer/watch_jobs_test.go

```diff
@@ -40,6 +40,34 @@ func testNodes(t *testing.T, state *state.StateStore) (drainingNode, runningNode
 	return n1, n2
 }
 
+func testNodesMultiDrain(t *testing.T, state *state.StateStore) (drainingNode, otherDrainingNode, runningNode *structs.Node) {
+	n1 := mock.Node()
+	n1.Name = "draining"
+	n1.DrainStrategy = &structs.DrainStrategy{
+		DrainSpec: structs.DrainSpec{
+			Deadline: time.Minute,
+		},
+		ForceDeadline: time.Now().Add(time.Minute),
+	}
+	require.Nil(t, state.UpsertNode(structs.MsgTypeTestSetup, 100, n1))
```

Comment:
Let's not add new tests using `testify` over `shoenig/test`:

```suggestion
	must.NoError(t, state.UpsertNode(structs.MsgTypeTestSetup, 100, n1))
```